klasses/utils.py

```diff
@@ -150,3 +155,37 @@ def unrevoke_certificate(user, bootcamp_run):
         result["msg"] = "Valid user and bootcamp_run must be provided."
 
     return result
+
+
+def manage_user_certificate_blocking(users, block_state, bootcamp_run):
+    """Block users for getting certificates in all bootcamp run enrollments"""
+    result = {"updated": False}
+    if users:
+        run_enrollments = BootcampRunEnrollment.objects.filter(
+            user__email__in=users, bootcamp_run=bootcamp_run
+        )
+
+        if not run_enrollments:
+            result[
+                "msg"
+            ] = "Matching query does not exist for users{} for run:{} ".format(
+                users, bootcamp_run
+            )
+            return result
+
+        run_enrollments.update(user_certificate_is_blocked=block_state)
+
+        state = "blocked"
+        if not block_state:
+            state = "unblocked"
+
+        blocked_users_emails = [run.user.email for run in run_enrollments]
+        result.update(
+            {
+                "updated": True,
+                "msg": "User(s) with email in {} ".format(blocked_users_emails)
```

Comment:
```suggestion
                "msg": "User(s) with email in {} are ".format(blocked_users_emails)
```

Reply:
It can be a single user in list or multiple so I thought it is better to drop `is/are` from the message.